src/web/app/pages-instructor/student-profile/student-profile.component.ts

```diff
@@ -12,26 +11,20 @@ import { StudentProfile } from './student-profile';
   styleUrls: ['./student-profile.component.scss'],
 })
 export class StudentProfileComponent implements OnInit {
-
   Gender: typeof Gender = Gender; // enum
 
+  @Input() photoUrl: string = '/assets/images/profile_picture_default.png';
   @Input() studentProfile: StudentProfile | undefined;
 
-  private backendUrl: string = environment.backendUrl;
-
   constructor() { }
 
   ngOnInit(): void {
   }
 
   /**
-   * Construct the url for the profile picture from the given key.
+   * Sets the profile picture of a student as the default image.
    */
-  getPictureUrl(pictureKey: string): string {
-    if (!pictureKey) {
-      return '/assets/images/profile_picture_default.png';
-    }
-    return `${this.backendUrl}/webapi/students/profilePic?blob-key=${pictureKey}`;
+  setDefaultPic(): void {
+    this.photoUrl = '/assets/images/profile_picture_default.png';
```

Comment:
This strengthens what I said about extracting this as constant.

Reply:
Yes correct (used in a couple of places)